src/schema/objects/metadata.yaml

```diff
@@ -3392,6 +3392,24 @@ SubjectArtefactDescription:
     If this field is set to `"n/a"`, it will be interpreted as absence of major
     source of artifacts except cardiac and blinks.
   type: string
+TablePosition:
+  name: TablePosition
+  display_name: Table Position
+  description: |
+    The table position, relative to an implementation-specific reference point,
+    often the isocenter. Values must be an array (1x3) of three distances in
+    millimeters in absolute coordinates (world coordinates). The
+    absolute coordinates are respectively right, anterior and superior (RAS+)
```

Comment:
In general (eg [here](https://slicer.readthedocs.io/en/latest/user_guide/coordinate_systems.html)), we use XYZ for absolute coordinate and RLAPSI for patient coordinate. I'm wondering if we shouldn't use XYZ in the context of table position, since it is defined in the absolute coordinate.